{"id":545899912,"status":"success","stage":"stage-4","name":"library:ci-corn","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.655Z","started_at":"2020-05-10T19:58:50.629Z","finished_at":"2020-05-10T20:30:54.485Z","duration":1923.856407,"queued_duration":1.023309,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899912","artifacts":[{"file_type":"trace","size":364885,"filename":"job.log","file_format":null}],"runner":{"id":380987,"description":"shared-runners-manager-6.gitlab.com","ip_address":"34.74.239.13","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899911,"status":"success","stage":"stage-3","name":"library:ci-math-classes","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.614Z","started_at":"2020-05-10T19:50:14.722Z","finished_at":"2020-05-10T19:58:49.475Z","duration":514.753363,"queued_duration":0.976158,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899911","artifacts":[{"file_type":"trace","size":171237,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899910,"status":"success","stage":"stage-3","name":"library:ci-color","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.598Z","started_at":"2020-05-10T19:50:14.043Z","finished_at":"2020-05-10T20:00:38.910Z","duration":624.86723,"queued_duration":0.349357,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899910","artifacts":[{"file_type":"trace","size":207482,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899909,"status":"success","stage":"stage-2","name":"plugin:ci-relation_algebra","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.582Z","started_at":"2020-05-10T19:38:57.059Z","finished_at":"2020-05-10T19:45:57.449Z","duration":420.390231,"queued_duration":1.85006,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899909","artifacts":[{"file_type":"trace","size":271156,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899908,"status":"success","stage":"stage-2","name":"plugin:ci-quickchick","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.569Z","started_at":"2020-05-10T19:45:54.501Z","finished_at":"2020-05-10T19:52:15.998Z","duration":381.497303,"queued_duration":1.569083,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899908","artifacts":[{"file_type":"trace","size":230456,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899907,"status":"success","stage":"stage-2","name":"plugin:ci-perennial","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.556Z","started_at":"2020-05-10T19:45:54.455Z","finished_at":"2020-05-10T20:09:38.303Z","duration":1423.848592,"queued_duration":1.57656,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899907","artifacts":[{"file_type":"trace","size":70028,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899906,"status":"success","stage":"stage-2","name":"plugin:ci-paramcoq","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.543Z","started_at":"2020-05-10T19:38:57.003Z","finished_at":"2020-05-10T19:40:03.606Z","duration":66.603217,"queued_duration":1.843363,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899906","artifacts":[{"file_type":"trace","size":115222,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899905,"status":"success","stage":"stage-2","name":"plugin:ci-mtac2","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.528Z","started_at":"2020-05-10T19:38:56.898Z","finished_at":"2020-05-10T19:43:06.462Z","duration":249.564624,"queued_duration":1.789638,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899905","artifacts":[{"file_type":"trace","size":144430,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899904,"status":"success","stage":"stage-2","name":"plugin:ci-fiat_parsers","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.515Z","started_at":"2020-05-10T19:38:56.860Z","finished_at":"2020-05-10T19:56:50.782Z","duration":1073.92126,"queued_duration":1.802163,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899904","artifacts":[{"file_type":"trace","size":264289,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899903,"status":"success","stage":"stage-2","name":"plugin:ci-equations","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.500Z","started_at":"2020-05-10T19:38:56.824Z","finished_at":"2020-05-10T19:47:09.561Z","duration":492.736765,"queued_duration":1.81913,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899903","artifacts":[{"file_type":"trace","size":102436,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899902,"status":"success","stage":"stage-2","name":"plugin:ci-elpi","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.486Z","started_at":"2020-05-10T19:38:56.788Z","finished_at":"2020-05-10T19:43:04.965Z","duration":248.17691,"queued_duration":1.831866,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899902","artifacts":[{"file_type":"trace","size":378573,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899901,"status":"success","stage":"stage-2","name":"plugin:ci-coqhammer","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.472Z","started_at":"2020-05-10T19:38:56.686Z","finished_at":"2020-05-10T19:41:51.264Z","duration":174.578632,"queued_duration":1.775348,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899901","artifacts":[{"file_type":"trace","size":39915,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899900,"status":"success","stage":"stage-2","name":"plugin:ci-coq_dpdgraph","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.459Z","started_at":"2020-05-10T19:38:56.628Z","finished_at":"2020-05-10T19:41:49.321Z","duration":172.692891,"queued_duration":1.765622,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899900","artifacts":[{"file_type":"trace","size":45517,"filename":"job.log","file_format":null}],"runner":{"id":380987,"description":"shared-runners-manager-6.gitlab.com","ip_address":"34.74.239.13","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899899,"status":"success","stage":"stage-2","name":"plugin:ci-bignums","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.442Z","started_at":"2020-05-10T19:45:54.418Z","finished_at":"2020-05-10T19:50:13.555Z","duration":259.137466,"queued_duration":1.594164,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899899","artifacts":[{"file_type":"trace","size":50965,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899898,"status":"success","stage":"stage-2","name":"plugin:ci-aac_tactics","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.419Z","started_at":"2020-05-10T19:38:56.518Z","finished_at":"2020-05-10T19:44:24.000Z","duration":327.482001,"queued_duration":1.706065,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899898","artifacts":[{"file_type":"trace","size":75442,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899897,"status":"success","stage":"stage-2","name":"library:ci-vst","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.405Z","started_at":"2020-05-10T19:45:54.354Z","finished_at":"2020-05-10T20:53:51.030Z","duration":4076.676818,"queued_duration":1.58585,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899897","artifacts":[{"file_type":"trace","size":112407,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899896,"status":"success","stage":"stage-2","name":"library:ci-verdi-raft","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.391Z","started_at":"2020-05-10T19:45:54.298Z","finished_at":"2020-05-10T20:14:21.351Z","duration":1707.053138,"queued_duration":1.589784,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899896","artifacts":[{"file_type":"trace","size":176334,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899893,"status":"success","stage":"stage-2","name":"library:ci-unimath","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.378Z","started_at":"2020-05-10T19:45:54.098Z","finished_at":"2020-05-10T20:52:30.628Z","duration":3996.53021,"queued_duration":1.444293,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899893","artifacts":[{"file_type":"trace","size":207289,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899891,"status":"success","stage":"stage-2","name":"library:ci-tlc","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.365Z","started_at":"2020-05-10T19:38:56.459Z","finished_at":"2020-05-10T19:43:01.061Z","duration":244.602861,"queued_duration":1.701082,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899891","artifacts":[{"file_type":"trace","size":34681,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899890,"status":"success","stage":"stage-2","name":"library:ci-stdlib2","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.353Z","started_at":"2020-05-10T19:45:54.034Z","finished_at":"2020-05-10T19:47:12.885Z","duration":78.851452,"queued_duration":1.438742,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899890","artifacts":[{"file_type":"trace","size":73322,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899888,"status":"failed","stage":"stage-2","name":"library:ci-sf","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.331Z","started_at":"2020-05-10T19:38:56.384Z","finished_at":"2020-05-10T19:41:36.036Z","duration":159.652139,"queued_duration":1.676143,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"failure_reason":"script_failure","web_url":"https://gitlab.com/coq/coq/-/jobs/545899888","artifacts":[{"file_type":"trace","size":25995,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899887,"status":"success","stage":"stage-2","name":"library:ci-math-comp","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.318Z","started_at":"2020-05-10T19:45:53.813Z","finished_at":"2020-05-10T21:10:18.557Z","duration":5064.743856,"queued_duration":1.277801,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899887","artifacts":[{"file_type":"trace","size":484893,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899885,"status":"success","stage":"stage-2","name":"library:ci-iris-lambda-rust","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.304Z","started_at":"2020-05-10T19:45:53.606Z","finished_at":"2020-05-10T20:34:16.243Z","duration":2902.63746,"queued_duration":1.12872,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899885","artifacts":[{"file_type":"trace","size":267733,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899884,"status":"success","stage":"stage-2","name":"library:ci-hott","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.290Z","started_at":"2020-05-10T19:38:56.154Z","finished_at":"2020-05-10T19:52:57.513Z","duration":841.359088,"queued_duration":1.493337,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899884","artifacts":[{"file_type":"trace","size":117842,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899883,"status":"success","stage":"stage-2","name":"library:ci-geocoq","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.275Z","started_at":"2020-05-10T19:45:53.543Z","finished_at":"2020-05-10T20:21:37.365Z","duration":2143.82178,"queued_duration":1.130261,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899883","artifacts":[{"file_type":"trace","size":497470,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899882,"status":"success","stage":"stage-2","name":"library:ci-flocq","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.262Z","started_at":"2020-05-10T19:38:56.113Z","finished_at":"2020-05-10T19:48:15.857Z","duration":559.743564,"queued_duration":1.506374,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899882","artifacts":[{"file_type":"trace","size":59040,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899881,"status":"success","stage":"stage-2","name":"library:ci-fiat-crypto-legacy","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.247Z","started_at":"2020-05-10T19:45:53.504Z","finished_at":"2020-05-10T20:53:42.227Z","duration":4068.723294,"queued_duration":1.146914,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899881","artifacts":[{"file_type":"trace","size":534112,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899880,"status":"success","stage":"stage-2","name":"library:ci-fiat-crypto","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.233Z","started_at":"2020-05-10T19:45:53.444Z","finished_at":"2020-05-10T21:09:04.122Z","duration":4990.678685,"queued_duration":1.152297,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899880","artifacts":[{"file_type":"trace","size":463770,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899879,"status":"success","stage":"stage-2","name":"library:ci-fcsl-pcm","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.219Z","started_at":"2020-05-10T19:38:56.073Z","finished_at":"2020-05-10T19:43:11.760Z","duration":255.68668,"queued_duration":1.516955,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899879","artifacts":[{"file_type":"trace","size":101294,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899878,"status":"success","stage":"stage-2","name":"library:ci-cross-crypto","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.207Z","started_at":"2020-05-10T19:38:56.035Z","finished_at":"2020-05-10T20:16:09.545Z","duration":2233.510033,"queued_duration":1.532371,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899878","artifacts":[{"file_type":"trace","size":49232,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899877,"status":"success","stage":"stage-2","name":"library:ci-coquelicot","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.193Z","started_at":"2020-05-10T19:38:55.915Z","finished_at":"2020-05-10T19:44:05.681Z","duration":309.765676,"queued_duration":1.464633,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899877","artifacts":[{"file_type":"trace","size":131110,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899876,"status":"success","stage":"stage-2","name":"library:ci-compcert","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.180Z","started_at":"2020-05-10T19:38:55.878Z","finished_at":"2020-05-10T20:00:57.775Z","duration":1321.897268,"queued_duration":1.476717,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899876","artifacts":[{"file_type":"trace","size":4194361,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899875,"status":"success","stage":"stage-2","name":"library:ci-bedrock2","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.166Z","started_at":"2020-05-10T19:38:55.697Z","finished_at":"2020-05-10T21:48:48.372Z","duration":7792.675047,"queued_duration":1.348158,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899875","artifacts":[{"file_type":"trace","size":201425,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899874,"status":"success","stage":"stage-2","name":"library:ci-argosy","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.151Z","started_at":"2020-05-10T19:38:55.460Z","finished_at":"2020-05-10T19:42:46.863Z","duration":231.403047,"queued_duration":1.162256,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899874","artifacts":[{"file_type":"trace","size":29537,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899873,"status":"success","stage":"stage-2","name":"validate:quick","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.137Z","started_at":"2020-05-10T19:45:25.166Z","finished_at":"2020-05-10T19:47:40.134Z","duration":134.968108,"queued_duration":0.348025,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899873","artifacts":[{"file_type":"trace","size":63722,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-07-09T19:47:37.918Z","tag_list":[]}
{"id":545899872,"status":"success","stage":"stage-2","name":"validate:edge+flambda","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.121Z","started_at":"2020-05-10T19:45:53.094Z","finished_at":"2020-05-10T19:51:38.495Z","duration":345.400484,"queued_duration":0.860182,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899872","artifacts":[{"file_type":"trace","size":107433,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-07-09T19:51:35.897Z","tag_list":[]}
{"id":545899871,"status":"success","stage":"stage-2","name":"validate:base+32bit","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.106Z","started_at":"2020-05-10T19:38:01.215Z","finished_at":"2020-05-10T19:42:16.697Z","duration":255.482636,"queued_duration":0.774112,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899871","artifacts":[{"file_type":"trace","size":67602,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-07-09T19:42:15.509Z","tag_list":[]}
{"id":545899870,"status":"success","stage":"stage-2","name":"validate:base","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.089Z","started_at":"2020-05-10T19:38:55.422Z","finished_at":"2020-05-10T19:43:15.418Z","duration":259.995864,"queued_duration":1.175172,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899870","artifacts":[{"file_type":"trace","size":69078,"filename":"job.log","file_format":null}],"runner":{"id":380987,"description":"shared-runners-manager-6.gitlab.com","ip_address":"34.74.239.13","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-07-09T19:43:14.226Z","tag_list":[]}
{"id":545899869,"status":"success","stage":"stage-2","name":"test-suite:base+async","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":true,"created_at":"2020-05-10T19:27:31.075Z","started_at":"2020-05-10T19:38:55.369Z","finished_at":"2020-05-10T20:09:37.119Z","duration":1841.750148,"queued_duration":1.170444,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899869","artifacts":[{"file_type":"trace","size":93424,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899868,"status":"success","stage":"stage-2","name":"test-suite:edge:dune:dev","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.061Z","started_at":"2020-05-10T19:34:51.026Z","finished_at":"2020-05-10T19:48:57.704Z","duration":846.677238,"queued_duration":0.804127,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899868","artifacts":[{"file_type":"trace","size":193483,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899867,"status":"success","stage":"stage-2","name":"test-suite:edge+flambda","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.048Z","started_at":"2020-05-10T19:45:52.737Z","finished_at":"2020-05-10T20:03:15.935Z","duration":1043.198059,"queued_duration":0.557144,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899867","artifacts":[{"file_type":"trace","size":118373,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899866,"status":"success","stage":"stage-2","name":"test-suite:base+32bit","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.033Z","started_at":"2020-05-10T19:38:01.091Z","finished_at":"2020-05-10T19:56:18.205Z","duration":1097.113061,"queued_duration":0.701031,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899866","artifacts":[{"file_type":"trace","size":91884,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899865,"status":"success","stage":"stage-2","name":"test-suite:base","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.018Z","started_at":"2020-05-10T19:38:55.044Z","finished_at":"2020-05-10T19:59:03.571Z","duration":1208.526744,"queued_duration":0.924413,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899865","artifacts":[{"file_type":"trace","size":93360,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899864,"status":"success","stage":"stage-2","name":"doc:ml-api:odoc","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:31.001Z","started_at":"2020-05-10T19:34:50.989Z","finished_at":"2020-05-10T19:39:36.522Z","duration":285.533261,"queued_duration":0.814786,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899864","artifacts":[{"file_type":"trace","size":76384,"filename":"job.log","file_format":null}],"runner":{"id":380987,"description":"shared-runners-manager-6.gitlab.com","ip_address":"34.74.239.13","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-07-09T19:39:35.337Z","tag_list":[]}
{"id":545899863,"status":"success","stage":"stage-2","name":"doc:stdlib:dune","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.987Z","started_at":"2020-05-10T19:34:50.944Z","finished_at":"2020-05-10T19:39:31.446Z","duration":280.501435,"queued_duration":0.823525,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899863","artifacts":[{"file_type":"trace","size":20456,"filename":"job.log","file_format":null}],"runner":{"id":380987,"description":"shared-runners-manager-6.gitlab.com","ip_address":"34.74.239.13","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-07-09T19:39:29.926Z","tag_list":[]}
{"id":545899862,"status":"success","stage":"stage-2","name":"doc:refman:dune","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.973Z","started_at":"2020-05-10T19:34:50.899Z","finished_at":"2020-05-10T19:40:40.126Z","duration":349.226604,"queued_duration":0.830475,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899862","artifacts":[{"file_type":"trace","size":28033,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-07-09T19:40:38.713Z","tag_list":[]}
{"id":545899861,"status":"success","stage":"stage-2","name":"doc:refman","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.957Z","started_at":"2020-05-10T19:38:54.921Z","finished_at":"2020-05-10T19:45:42.586Z","duration":407.664454,"queued_duration":0.853582,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899861","artifacts":[{"file_type":"trace","size":1236180,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-07-09T19:45:41.392Z","tag_list":[]}
{"id":545899860,"status":"success","stage":"stage-1","name":"plugin:plugin-tutorial","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.928Z","started_at":"2020-05-10T19:27:34.224Z","finished_at":"2020-05-10T19:31:27.563Z","duration":233.339271,"queued_duration":1.948312,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899860","artifacts":[{"file_type":"trace","size":57804,"filename":"job.log","file_format":null}],"runner":{"id":44949,"description":"shared-runners-manager-4.gitlab.com","ip_address":"13.37.237.197","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899859,"status":"failed","stage":"stage-1","name":"test-suite:edge+4.11+trunk+dune","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":true,"created_at":"2020-05-10T19:27:30.916Z","started_at":"2020-05-10T19:27:34.187Z","finished_at":"2020-05-10T20:08:13.859Z","duration":2439.671502,"queued_duration":1.928502,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"failure_reason":"script_failure","web_url":"https://gitlab.com/coq/coq/-/jobs/545899859","artifacts":[{"file_type":"trace","size":174483,"filename":"job.log","file_format":null}],"runner":{"id":380987,"description":"shared-runners-manager-6.gitlab.com","ip_address":"34.74.239.13","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-05-24T20:08:12.711Z","tag_list":[]}
{"id":545899858,"status":"success","stage":"stage-1","name":"pkg:nix","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.904Z","started_at":"2020-05-10T19:27:34.151Z","finished_at":"2020-05-10T20:07:33.796Z","duration":2399.644384,"queued_duration":1.908361,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899858","artifacts":[{"file_type":"trace","size":572864,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899857,"status":"success","stage":"stage-1","name":"pkg:opam","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.892Z","started_at":"2020-05-10T19:27:34.092Z","finished_at":"2020-05-10T19:50:04.970Z","duration":1350.877789,"queued_duration":1.861998,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899857","artifacts":[{"file_type":"trace","size":20445,"filename":"job.log","file_format":null}],"runner":{"id":380987,"description":"shared-runners-manager-6.gitlab.com","ip_address":"34.74.239.13","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":null,"tag_list":[]}
{"id":545899856,"status":"success","stage":"stage-1","name":"lint","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.881Z","started_at":"2020-05-10T19:27:34.019Z","finished_at":"2020-05-10T19:30:45.216Z","duration":191.196556,"queued_duration":1.802948,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899856","artifacts":[{"file_type":"trace","size":51325,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":null,"tag_list":[]}
{"id":545899855,"status":"failed","stage":"stage-1","name":"windows32","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.865Z","started_at":"2020-05-10T19:29:13.381Z","finished_at":"2020-05-11T00:00:01.254Z","duration":16247.872685,"queued_duration":101.179624,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"failure_reason":"stuck_or_timeout_failure","web_url":"https://gitlab.com/coq/coq/-/jobs/545899855","artifacts":[{"file_type":"trace","size":304004,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-05-17T22:37:38.149Z","tag_list":["windows-inria"]}
{"id":545899852,"status":"success","stage":"stage-1","name":"windows64","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.850Z","started_at":"2020-05-10T19:29:06.417Z","finished_at":"2020-05-10T23:25:35.166Z","duration":14188.749535,"queued_duration":94.229981,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899852","artifacts":[{"file_type":"trace","size":463917,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-05-17T23:25:33.743Z","tag_list":["windows-inria"]}
{"id":545899850,"status":"success","stage":"stage-1","name":"build:quick","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":true,"created_at":"2020-05-10T19:27:30.837Z","started_at":"2020-05-10T19:27:33.930Z","finished_at":"2020-05-10T19:45:24.687Z","duration":1070.757073,"queued_duration":1.757996,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899850","artifacts":[{"file_type":"trace","size":193387,"filename":"job.log","file_format":null}],"runner":{"id":44028,"description":"shared-runners-manager-3.gitlab.com","ip_address":"35.196.21.178","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-05-17T19:45:23.211Z","tag_list":[]}
{"id":545899849,"status":"success","stage":"stage-1","name":"build:base+async","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":true,"created_at":"2020-05-10T19:27:30.824Z","started_at":"2020-05-10T19:27:33.877Z","finished_at":"2020-05-10T20:36:50.392Z","duration":4156.515399,"queued_duration":1.728629,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899849","artifacts":[{"file_type":"trace","size":278542,"filename":"job.log","file_format":null}],"runner":{"id":380986,"description":"shared-runners-manager-5.gitlab.com","ip_address":"34.74.209.140","active":true,"paused":false,"is_shared":true,"runner_type":"instance_type","name":"gitlab-runner","online":false,"status":"stale"},"artifacts_expire_at":"2020-05-17T20:36:48.907Z","tag_list":[]}
{"id":545899847,"status":"success","stage":"stage-1","name":"build:edge+flambda:dune:dev","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.808Z","started_at":"2020-05-10T19:27:33.673Z","finished_at":"2020-05-10T19:34:49.934Z","duration":436.261637,"queued_duration":1.540659,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899847","artifacts":[{"file_type":"trace","size":51973,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-05-17T19:34:47.994Z","tag_list":[]}
{"id":545899846,"status":"success","stage":"stage-1","name":"build:edge+flambda","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.795Z","started_at":"2020-05-10T19:27:33.344Z","finished_at":"2020-05-10T19:45:52.036Z","duration":1098.691094,"queued_duration":1.228395,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899846","artifacts":[{"file_type":"trace","size":311647,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-05-17T19:45:49.580Z","tag_list":[]}
{"id":545899845,"status":"success","stage":"stage-1","name":"build:base+32bit","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.782Z","started_at":"2020-05-10T19:27:32.968Z","finished_at":"2020-05-10T19:38:00.262Z","duration":627.293844,"queued_duration":0.866361,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899845","artifacts":[{"file_type":"trace","size":302492,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-05-17T19:37:58.380Z","tag_list":[]}
{"id":545899844,"status":"success","stage":"stage-1","name":"build:base","ref":"pr-12294","tag":false,"coverage":null,"allow_failure":false,"created_at":"2020-05-10T19:27:30.768Z","started_at":"2020-05-10T19:27:32.560Z","finished_at":"2020-05-10T19:38:53.905Z","duration":681.345337,"queued_duration":0.473516,"user":{"id":2256237,"username":"coqbot","name":"coqbot","state":"active","avatar_url":"https://gitlab.com/uploads/-/system/user/avatar/2256237/avatar.png","web_url":"https://gitlab.com/coqbot","created_at":"2018-04-28T07:53:57.210Z","bio":"Hello, I'm a bot.","location":"","public_email":"","skype":"","linkedin":"","twitter":"","website_url":"","organization":"","job_title":"","pronouns":null,"bot":false,"work_information":null,"followers":0,"following":0,"local_time":null},"commit":{"id":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","short_id":"7ea8894b","created_at":"2020-05-10T21:27:14.000+02:00","parent_ids":["f40839ee880ac3e79d560e26f55f462f8ffd2e05"],"title":"[errors] Don't raise inside exception printers","message":"[errors] Don't raise inside exception printers\n\nThis is wrong and not properly handled anymore, instead the handlers\nare expected to return the error message.\n\nThis is a backport of #11490 , as it turns out this doesn't make sense\nin 8.11 neither.\n\nFixes #12291\n","author_name":"Emilio Jesus Gallego Arias","author_email":"e+git@x80.org","authored_date":"2020-05-10T20:42:16.000+02:00","committer_name":"Emilio Jesus Gallego Arias","committer_email":"e+git@x80.org","committed_date":"2020-05-10T21:27:14.000+02:00","trailers":{},"web_url":"https://gitlab.com/coq/coq/-/commit/7ea8894b7847cadfd3f022bdeac699e92149b3a1"},"pipeline":{"id":144557643,"iid":15990,"project_id":6138686,"sha":"7ea8894b7847cadfd3f022bdeac699e92149b3a1","ref":"pr-12294","status":"failed","source":"push","created_at":"2020-05-10T19:27:30.752Z","updated_at":"2020-05-11T00:00:01.511Z","web_url":"https://gitlab.com/coq/coq/-/pipelines/144557643"},"web_url":"https://gitlab.com/coq/coq/-/jobs/545899844","artifacts":[{"file_type":"trace","size":351029,"filename":"job.log","file_format":null}],"runner":null,"artifacts_expire_at":"2020-05-17T19:38:51.661Z","tag_list":[]}
